src/components/base/loader/index.jsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import propTypes from "prop-types";
+import { connect } from "react-redux";
+import Image from "components/base/image";
+import { ALT_LOADER_TAG, ICON_LOADER } from "utils/const";
+import styles from "components/base/loader/index.module.scss";
+
+function Loader({ imageClassName, isLoading }) {
+  if (!isLoading) return null;
```

Comment:
Use consistent brace. //see https://eslint.org/docs/latest/rules/curly
```suggestion
  if (!isLoading) {
     return null
  };
```